### PR TITLE
Inital spike for setting up the gem. It includes a rename because I realized I didn't need to depend on dalli

### PR DESCRIPTION
**Pull request by anfleene**

### CODE REVIEW

File: breaker-rails-cache-repo.gemspec [hunk 1]
```diff
@@ -14,16 +14,16 @@ Gem::Specification.new do |spec|
...
   spec.summary       = %q{This gem is a memcached implementation of the repo pattern used by the breaker gem}
-  spec.description   = %q{Dalli based repo for the breaker gem}
+  spec.description   = %q{Rails Cache backed repo for the breaker gem}
   spec.license       = "MIT"
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
-  spec.add_dependency "dalli"
   spec.add_dependency "breaker"
```
Comment (bbergstrom) [inline breaker-rails-cache-repo.gemspec]: Did you consider locking down breaker to the major version you wrote your extension against?

**Comment by bbergstrom** [comment]
CR :+1:

**Comment by anfleene** [comment]
@bbergstrom I think this is a fulling working version. I'm going to start bundling this stuff up.

**Comment by bbergstrom** [comment]
:shipit:

**Pull request merged**